packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -326,7 +326,10 @@ type BaseInputCommonProps = FormInputLabelProps &
      * @default true
      **/
     isTableInputCell?: boolean;
-
+    /**
+     * Callback to be invoked when the backdrop is clicked
+     */
+    onBackdropClick?: () => void;
```

Comment:
Should we call this `onInputWrapperClick`? that way its clear on which element we're calling it. Backdrop is a bit misleading because in other places we use it in the context of faded overlay when modal / dropdown opens